### ISSUE TITLE
Underwater plants become water source blocks

### ISSUE THREAD

**Issue by ThatOneMendel** [Jacky1356400/SimpleSponge#20]
heya.

When I place down a vanilla sponge it destroys all the plants nearby (seaweed, kept, etc)

But when I use this sponge they remain, and become water source blocks. I have to destroy the plant before I can get the sponge to clear the water source.

Is this a bug or a feature?

**Comment by JackyyTV** [comment]
Most likely a bug, because when I first started porting this mod from 1.12 over to 1.13+, I added a bit of code that accounts for waterlogged blocks (i.e. non-full blocks with water in them) to prevent them from having water:
https://github.com/JackyyTV/SimpleSponge/blob/34ca1ade54ab75827bd814357f6eb362302f3fa2/src/main/java/jackyy/simplesponge/block/BlockSpongeBase.java#L76-L81
I guess I didn't account for the underwater plants as well when I did that, so now this happened. I'll just make the sponges behave closer to vanilla ones with handling the underwater plants.
